features/regexp-static-properties.yml

```
@@ -0,0 +1,13 @@
+name: RegExp static properties
+description: The `RegExp` object has several static properties that are the input and results of the most-recent regular expression match.
```

```suggestion
description: The `RegExp` object has several static properties used to access the input and results of the most-recent regular expression match.
```

I somehow feel like this is more natural to read.

---

Yeah, that's fair. We can tighten it up a little though:

```suggestion
description: The `RegExp` object has several static properties to access the input and results of the most-recent regular expression match.
```